libbeat/docs/shared-systemd.asciidoc

```diff
@@ -0,0 +1,103 @@
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+systemctl enable {beatname_lc}
+------------------------------------------------
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+systemctl disable {beatname_lc}
+------------------------------------------------
+
+
+==== {beatname_uc} status and logs
+
+To get the service status, use `systemctl`:
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+systemctl status {beatname_lc}
+------------------------------------------------
+
+Logs are stored by default in journald. To view the Logs, use `journalctl`:
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+journalctl -u {beatname_lc}.service
+------------------------------------------------
+
+NOTE: The unit file included in the packages sets the `-e` flag by default.
+This flag makes {beatname_uc} log to stderr and disables other log outputs.
+Systemd stores all output sent to stderr in journald.
+
+[float]
+=== Customize systemd unit for {beatname_uc}
+
+The systemd service unit file includes environment variables that you can
+override to change the default options.
+
+[cols="<h,<,<m",options="header",]
+|=======================================
+| Variable | Description | Default value
+| BEAT_LOG_OPTS | Log options | `-e`
+| BEAT_CONFIG_OPTS | Flags for configuration file path | `-c /etc/{beatname_lc}/{beatname_lc}.yml`
+| BEAT_PATH_OPTS | Other paths | -path.home /usr/share/{beatname_lc} -path.config /etc/{beatname_lc} -path.data /var/lib/{beatname_lc} -path.logs /var/log/{beatname_lc}
+|=======================================
+
+To override these variables, create a drop-in unit file in the
++/etc/systemd/system/{beatname_lc}.service.d+ directory.
+
+For example a file with the following content placed in
++/etc/systemd/system/{beatname_lc}.service.d/debug.conf+
+would override `BEAT_LOG_OPTS` to enable debug for Elasticsearch output.
+
+["source", "systemd", subs="attributes"]
+------------------------------------------------
+[Service]
+Environment=BEAT_LOG_OPTS="-e -d elasticsearch"
+------------------------------------------------
+
+To use settings from the {beatname_uc} file, empty the environment
+variable. For example:
+
+["source", "systemd", subs="attributes"]
+------------------------------------------------
+[Service]
+Environment=BEAT_LOG_OPTS=""
+------------------------------------------------
+
+To apply your changes, reload the systemd configuration and restart
+the service:
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+systemctl daemon-reload
+systemctl restart {beatname_lc}
+------------------------------------------------
+
+NOTE: It is recommended that you use a configuration management tool to
+include drop-in unit files. If you need to add a drop-in manually, use
+`systemctl edit {beatname_lc}.service`.
```

Comment:
Change:

```
`systemctl edit {beatname_lc}.service`
```

To

```
+systemctl edit {beatname_lc}.service+
```

Reply:
Oops, true, you already mentioned that, thanks!
I have wrapped a couple more command lines that were unwrapped.